pontoon/urls.py

```diff
@@ -40,6 +41,9 @@
     url(r'^pontoon\.js$',
         RedirectView.as_view(url='/static/js/pontoon.js', permanent=True)),
 
+    # Robots.txt
+    (r'^robots.txt$', lambda r: HttpResponse("User-agent: *\nDisallow: /a/\nDisallow: /admin/", content_type="text/plain")),
```

Comment:
BTW, @Osmose, which one will be better, rendering in views or using direct_to_template in urls? The only extra line in views will be logging I guess.